docs/features/prompts/quick-start.mdx

```diff
@@ -85,20 +85,20 @@ You must be using Helicone via Proxy. If you are not, please refer to our [Quick
       </Step>
       <Step title="Replace any text input">
 
-      Using the backtick string formatter in JavaScript, add `hprompt` in front of your backtick to automatically format your text so that Helicone can determine where your variables are.
+      Using the backtick string formatter in JavaScript, add `hprompt()` in front of your backtick to automatically format your text so that Helicone can determine where your variables are.
```

Comment:
```suggestion
      Using the backtick string formatter in JavaScript, add `hprompt` in front of your backtick to automatically format your text so that Helicone can determine where your variables are.
```